From 972bdf6aefe414f215683acab6ab21734aee2511 Mon Sep 17 00:00:00 2001
From: xuhuicong <xhc@rock-chips.com>
Date: Thu, 28 Apr 2016 15:22:45 +0800
Subject: [PATCH] video: rockchip: hdmi: misc clean up to hdmi driver

remove unused dts properties parsed code and print an error message
when enable hdmi clk error

Change-Id: I92f37f5c1dc2cd8dbf18744f4fd17a52bc25080f
Signed-off-by: xuhuicong <xhc@rock-chips.com>
---
 .../hdmi/rockchip-hdmiv2/rockchip_hdmiv2.c         | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/drivers/video/rockchip/hdmi/rockchip-hdmiv2/rockchip_hdmiv2.c b/drivers/video/rockchip/hdmi/rockchip-hdmiv2/rockchip_hdmiv2.c
index 2b04162fd02d..4c244c2367d8 100644
--- a/drivers/video/rockchip/hdmi/rockchip-hdmiv2/rockchip_hdmiv2.c
+++ b/drivers/video/rockchip/hdmi/rockchip-hdmiv2/rockchip_hdmiv2.c
@@ -482,17 +482,6 @@ static int rockchip_hdmiv2_parse_dt(struct hdmi_dev *hdmi_dev)
 		syscon_regmap_lookup_by_phandle(np, "rockchip,grf");
 	if (IS_ERR(hdmi_dev->grf_base)) {
 		hdmi_dev->grf_base = NULL;
-	} else {
-		if (of_property_read_u32(np, "rockchip,grf_reg_offset",
-					 &hdmi_dev->grf_reg_offset)) {
-			pr_err("get cru_reg_offset failed\n");
-			return -ENXIO;
-		}
-		if (of_property_read_u32(np, "rockchip,grf_reg_shift",
-					 &hdmi_dev->grf_reg_shift)) {
-			pr_err("get cru_reg_shift failed\n");
-			return -ENXIO;
-		}
 	}
 	#endif
 	return 0;
@@ -558,6 +547,7 @@ static int rockchip_hdmiv2_probe(struct platform_device *pdev)
 	pm_runtime_enable(hdmi_dev->dev);
 	/*enable pd and pclk and hdcp_clk*/
 	if (rockchip_hdmiv2_clk_enable(hdmi_dev) < 0) {
+		dev_err(&pdev->dev, "failed to enable hdmi clk\n");
 		ret = -ENXIO;
 		goto failed1;
 	}
@@ -699,7 +689,7 @@ static int rockchip_hdmiv2_probe(struct platform_device *pdev)
 	kfree(hdmi_dev->phy_table);
 	kfree(hdmi_dev);
 	hdmi_dev = NULL;
-	dev_err(&pdev->dev, "rk3288 hdmi probe error.\n");
+	dev_err(&pdev->dev, "rockchip hdmiv2 probe error.\n");
 	return ret;
 }
 
-- 
2.35.3

